test/features/user_records_an_application_test.rb

```diff
@@ -37,6 +37,21 @@ def test_an_application_with_no_company_is_rejected
     assert page.has_field? "application_company"
   end
 
+  def test_an_application_with_no_url_is_accepted
+    user = create(:person)
+    page.set_rack_session(user_id: user.id)
+    visit dashboard_path
+    click_link_or_button "new_application"
+    fill_in "application_company", with: "Google"
+    fill_in "application_location", with: "San Francisco, CA"
+    fill_in "application_applied_on", with: Date.today
+    select "in_progress", from: "application_status"
+
```

Comment:
the test should have two spaces. the top section should have the set up for the test. this is lines 41-43. then a space. then all the actions for the test. then a space. then the assertions